CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.22.4] - 15-Jan-2021
```

Comment:
I would update the version to 0.23.0 as it's not compatible with the old forked sdk and server. Update the pakage.json too.

Reply:
0.22.3 yeah

Reply:
Makes sense. If I put `0.23.0` to the package.json, will the release task on Travis change it to `0.23.1` ?

Reply:
Travis release trigger on main account will release and tag version  '0.23.0'. After that, it will upgrade the version to '0.23.1' but for the next release.